xml/System.Data.Common/DbCommand.xml

```diff
@@ -144,7 +144,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>When overridden in a derived class, attempts to cancel the execution of a <see cref="T:System.Data.Common.DbCommand" />.</summary>
+        <summary>Attempts to cancel the execution of a <see cref="T:System.Data.Common.DbCommand" />.</summary>
```

Comment:
The original wording is correct here and for all the other abstract members of this class, according to [the guidelines](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Abstract-and-virtual-member). Why do you think it should be removed?

Reply:
This is IMHO a general issue with the System.Data.Common docs. Yes, DbCommand is abstract, but IMHO the docs are intended for users of DbCommand implementations, rather than for writers of System.Data providers. In other words, an end-user doesn't really care about whether Cancel is implemented in DbCommand or in implementations - they're just looking for docs on what the method does.

Another way to look at it, is that almost all System.Data methods are abstract and need to be overridden by providers; but I don't think we want to prefix all method docs with "When overridden in a derived class".

I do think we should include provider-writer-facing notes in the remarks, but at the very end and with a clear indication that they're not meant for users.

What do you think?

Reply:
I see your point. Do you think we should update the guidelines then? I don't think this is specific to Sytem.Data.Common docs. I could see having the usual description for the method, and then perhaps having generic text in the Remarks section like you suggested. Similar to what we have in the [guidelines for abstract classes](https://github.com/dotnet/dotnet-api-docs/wiki/Remarks#abstract-class). cc @BillWagner @mairaw

Reply:
> Do you think we should update the guidelines then?

Yeah, maybe... The thing is, not all abstract classes necessarily represent the same case - System.Data.Common really is a provider abstraction where we have two consumers (end users vs. provider implementors); other cases of abstract classes may be slightly different. But yeah, it's something to think about.